sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkTest.java

```diff
@@ -247,76 +239,6 @@ void builder_addSamplerCustomizer() {
         .isEqualTo(sampler2);
   }
 
-  @Test
-  void builder_addSpanExporterCustomizer() {
```

Comment:
Test logic in here moves to `FullConfigTest`. Its very difficult to test exporter and processor customizers in this test suite which excludes all exporter implementation. Better suited for an end-to-end style test.

Reply:
This is a small bummer to see go away. I think maybe it was previously testing more than it needed to...but I still think it's nice to have a _unit_ test that confirms the contract of the builder with the customizer(s)...or at least that `build()` invokes the customizer....but we do have coverage elsewhere, so that's something, it's just higher up on the testing pyramid.

Reply:
`FullConfigTest` does assert that, just in a less indirect way. I'm disappointed too, but these tests were taking advantage of weird mechanic of calling the exporter customizer even when the exporter is set to `none`. With that mechanic removed, they can't exist in this simple form. 🙁